server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -48,11 +49,17 @@ export function create(
         throttleIdSuffix: Constants.alfredRestThrottleIdSuffix,
     };
 
+    // Throttling logic for creating documents to provide per-cluster rate-limiting at the HTTP route level
+    const createDocThrottleOptions: Partial<IThrottleMiddlewareOptions> = {
```

Comment:
now that you have renamed both throttlers, it might be better to rename the options with similar naming pattern as well. One is per tenant and the other is per cluster.

Reply:
Got it. Done.